YACS-RCOS/yacs.n
# loading spinner centering

**Issue**
Closes #280 
Loading spinner is not centered and footer wasn't remained at the page bottom

**Database Changes/Migrations**
N/A

**Test Modifications**

N/A

**Test Procedure**

run `docker volume rm yacsn_postgres_data` to clear database, or manually stop yacs_api in docker dashboard.
Navigate to explore page
Observe loading spinner and footer position 

**Photos**
Before
![image](https://user-images.githubusercontent.com/43757365/98628252-0145f700-22cb-11eb-9231-db65b9130d81.png)

After
![image](https://user-images.githubusercontent.com/43757365/98628237-fa1ee900-22ca-11eb-93b6-1c16aa8defea.png)


**Additional Info**

N/A
